kyuubi-main/src/main/scala/org/apache/kyuubi/engine/ProcBuilder.scala

```diff
@@ -158,4 +134,25 @@ object ProcBuilder {
     }
   }
 
+  private def getRollAppendProcessLogFile(
+      workingDir: Path, module: String, processLogRetainTimeMillis: Long): File = synchronized {
+    val currentTime = System.currentTimeMillis()
+    val processLogPath = workingDir
+    val totalExistsFile = processLogPath.toFile.listFiles()
+    val sorted = totalExistsFile.sortBy(_.getName.split("\\.").last.toInt)
```

Comment:
this will throw an exception when the file is an invalid file

It's better to list files only with a filter that starts w/ `module` and skip those files that are invalid

Reply:
update it.